src/components/global/navigation/navbarMobile.tsx

```diff
@@ -118,42 +188,69 @@ export function NavbarMobile({ dark, links }: NavbarMobileProps) {
   };
 
   return (
-    <StyledNavMobile isMenuOpen={isMenuOpen} dark={dark}>
-      <StyledBtnMobile onClick={() => setIsMenuOpen(!isMenuOpen)}>
-        {isMenuOpen ? (
-          <Image
-            src={"/svg/icon-close.svg"}
-            alt="close icon"
-            style={iconStyleMenu}
-            width={24}
-            height={24}
-          />
-        ) : (
-          <Image
-            src={"/svg/icon-burger.svg"}
-            alt="burger icon"
-            style={iconStyleMenu}
-            width={24}
-            height={24}
-          />
-        )}
-      </StyledBtnMobile>
-      <StyledWrapperMobile isMenuOpen={isMenuOpen}>
-        {links.map((link, index) => (
-          <StyledDivMobile key={index} index={index}>
-            <StyledLinkMobile href={`/${link.path}`} dark={dark}>
-              {link.icon && (
-                <Image src={link.icon} alt={link.name} style={iconStyle} />
-              )}
-              {!link.icon && link.name}
-            </StyledLinkMobile>
-            <StyledNavSpan
-              isActive={pathname === `/${link.path}`}
-              dark={dark}
+    <>
+      <StyledImage dark={dark}>
```

Comment:
I see `StyledImage` is actually a div. Therefore it should be called something like `StyledImageWrapper` etc.